### PR TITLE
LRQA-73533

### PR DESCRIPTION
**Pull request by sandrodw3**
cc @Tim-Cao @david-gutierrez-mesa

### CODE REVIEW

**Comment by sandrodw3** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdminWithPerformance.testcase [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			if (("${responseTime}" == "0") || ("${renderTime}" == "0") || ("${loadTime}" == "0")) {
+				fail("Test is required to update");
+			}
+
+			var result1 = MathUtil.isLessThan("${responseTime}", "2000");
+			var result2 = MathUtil.isLessThan("${renderTime}", "2000");
+			var result3 = MathUtil.isLessThan("${loadTime}", "4000");
+
+			if (("${result1}" == "false") || ("${result2}" == "false") || ("${result3}" == "false")) {
+				fail("Performance is required to improve");
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdminWithPerformance.testcase:55]: If this test fails, will we know which variable was the one that make it fail?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdminWithPerformance.testcase [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+	@description = "This is a use case for LPS-123587. Test the loading time when admin visit the Pages admin with hundreds of pages."
+	@priority = "5"
+	test TestLoadingTimeWhenHundredsOfPagesInSite {
+		task ("Add five hundred pages via Groovy script") {
+			ServerAdministration.openServerAdmin();
+
+			var script = TestCase.getFileContent(fileName = "AddLayouts.groovy");
+
+			ServerAdministration.executeScript(
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdminWithPerformance.testcase:26]: Is there a way to pass parameters here? just in order to make this method more generic

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/dependencies/AddLayouts.groovy [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.service.GroupLocalServiceUtil;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.PortalUtil;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.service.UserLocalServiceUtil;
+import com.liferay.petra.string.StringPool;
+
+class AddLayouts {
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/dependencies/AddLayouts.groovy:16]: If the number of pages is hardcoded to 500 maybe we should change the name to Add500Layouts

**Comment by david-gutierrez-mesa** [comment]
I reviewed the code and in general, it's fine but IMHO this is not a performance test at all. Pohsi is not the way of doing this type of test.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdminWithPerformance.testcase (hunk 1 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdminWithPerformance.testcase:55]: There is a report printed in console before assertion
https://github.com/liferay-echo/liferay-portal/pull/7398/commits/ec43a4863a00e76803efb61fc9481b0c851f28b1#diff-cecb191518a5b23c5a8ec1096c5c645408eae37be467e13baefaf3c60b3d219dR40-R44

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdminWithPerformance.testcase (hunk 2 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/PagesAdminWithPerformance.testcase:26]: We cannot modify this macro but we could override the groovy script in our case.

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**